common.sh

```diff
@@ -437,6 +437,7 @@ EOF
 alinux_install_deps() {
     cat <<-"EOF" >> ${tmp_script}
     sudo yum -y groupinstall 'Development Tools'
+    sudo yum install -y gcc-gfortran
```

Comment:
I think `Development Tools` include gcc and gfortran
```
[ec2-user@ip-172-31-11-68 SubspaceBenchmarks-ompi]$ yum groupinfo "Development Tools"
Loaded plugins: dkms-build-requires, priorities, update-motd, upgrade-helper
1067 packages excluded due to repository priority protections

Group: Development tools
 Group-Id: development
 Description: A basic development environment.
 Mandatory Packages:
   =autoconf
   =automake
    binutils
   =bison
   =flex
   =gcc
   =gcc-c++
   =gdb
   =gettext
   =kexec-tools
   =latrace
   =libtool
    make
   =patch
    pkgconfig
   =rpm-build
   =strace
   =system-rpm-config
   =systemtap-runtime
 Default Packages:
   =byacc
   =crash
   =cscope
   =ctags
   =cvs
   =diffstat
   =doxygen
   =elfutils
   =gcc-gfortran
   =git
   =indent
   =intltool
   =ltrace
   =patchutils
   =rcs
   =subversion
   =swig
   =systemtap
   =texinfo
   =valgrind
 Optional Packages:
   ant
   bzr-python27
   ccache
   chrpath
   clips
   clips-devel
   clips-doc
   clips-emacs
   clips-xclips
   clipsmm-devel
   clipsmm-doc
   cmake
   cmucl
   colordiff
   cvsps
   darcs
   dejagnu
   email2trac
   expect
   ftnchek
   gcc-gnat
   ghc
   git
   glibc-utils
   haskell-platform
   imake
   jpackage-utils
   lslk
   lua
   mercurial
   mercurial-python27
   mock
   mod_dav_svn
   nasm
   nqc
   nqc-doc
   ocaml
   perltidy
   python27-babel
   python27-docs
   qgit
   rpmdevtools
   rpmlint
   sbcl
   systemtap-initscript
   trac
   trac-git-plugin
   trac-mercurial-plugin
   trac-webadmin
   translate-toolkit
```

Reply:
Yeah they do, (I knew that) I just wanted to be consistent. I can remove it.